pkgs/by-name/an/antora-ui-default/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  fetchFromGitLab,
+  lib,
+  stdenvNoCC,
+}:
+let
+  srcFetchFromGitLab = {
+    hash = "sha256-q2FwkwzjanxTIxjMpCyMpzPt782uYZiWVdZ7Eev79oM=";
+    owner = "trueNAHO";
+    repo = "antora-ui-default";
+    rev = "83bf9bf5f22a6dee397f8b089eb0315c14a278b5";
+  };
+in
+stdenvNoCC.mkDerivation {
+  installPhase = ''install --mode 755 -D "$src/ui-bundle.zip" "$out"'';
+
+  meta = {
+    description = "Antora default UI bundle";
+    homepage = "https://gitlab.com/antora/antora-ui-default";
+    license = lib.licenses.mpl20;
+
+    longDescription = ''
+      > A UI bundle is a [ZIP
+      > archive](https://en.wikipedia.org/wiki/Zip_(file_format)) or directory
+      > that contains one or more UIs for a site.
+      >
+      > -- Antora
+      >    https://docs.antora.org/antora/3.1/playbook/ui-bundle-url
+
+      This UI bundle is intended to be passed to `antora`'s `--ui-bundle-url`
+      flag or injected into the [`ui.bundle.url`
+      key](https://docs.antora.org/antora/3.1/playbook/ui-bundle-url/#url-key)
+      to avoid irreproducible
+      [`https://gitlab.com/antora/antora-ui-default/-/jobs/artifacts/HEAD/raw/build/ui-bundle.zip?job=bundle-stable`](https://gitlab.com/${srcFetchFromGitLab.owner}/${srcFetchFromGitLab.repo}/-/blob/${srcFetchFromGitLab.rev}/README.adoc#user-content-use-the-default-ui)
+      references.
+    '';
+
+    maintainers = [ lib.maintainers.naho ];
+    platforms = lib.platforms.all;
+  };
+
+  phases = [ "installPhase" ];
+  pname = srcFetchFromGitLab.repo;
```

Comment:
No need to indirect here. Just write out the name. https://github.com/NixOS/nixpkgs/issues/277994